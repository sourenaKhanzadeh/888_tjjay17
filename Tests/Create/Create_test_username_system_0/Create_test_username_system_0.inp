Enter user type: AA
Login 
Admin
Enter new username: George
Logout
