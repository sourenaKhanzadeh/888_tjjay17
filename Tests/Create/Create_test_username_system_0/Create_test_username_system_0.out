Welcome to the add credit page
Logged in as admin
New user name: George
The username does not exist on the database
Session terminated
